packages/DatePicker/src/helpers.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+export function getExtendedInputProps(children) {
```

Comment:
I'm calling this `extractChildren` in the SidePanel and I think Alex suggested the same name in the ListBox PR so lets keep the same name?